packages/lit-dev-content/site/docs/components/shadow-dom.md

```diff
@@ -205,37 +205,48 @@ render() {
 
 For more information, see [HTMLSlotElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement) on MDN.
 
-### @queryAssignedNodes decorator { #query-assigned-nodes }
+### @queryAssignedElements and @queryAssignedNodes decorator { #query-assigned-elements }
```

Comment:
Maybe we should keep the fragment as `query-assigned-nodes`, so that bookmarks don't break? Otherwise maybe just `query-assigned` if we're ok with changing it.

(Also TIL you could change the fragment like that!)

Reply:
Great point. Will do.

Reply:
Done!